deploy/deploy_marketplace.sh

```diff
@@ -178,6 +179,10 @@ EOF
     echo "Give OLM 60 seconds to process the installplan..."
     sleep 60
 
+    if [ "$KVM_EMULATION" = true ]; then
+      oc set env -n ${TARGET_NAMESPACE} deployment/hco-operator --containers='hyperconverged-cluster-operator' KVM_EMULATION='true'
```

Comment:
Isn't OLM going to overwrite any changes you make to the operator?

Reply:
yes, you right: after a certain amount of time the pod got recreated without that value

Reply:
Any other idea?

Reply:
probably this one: https://github.com/operator-framework/operator-lifecycle-manager/blob/master/doc/contributors/design-proposals/operator-config.md